src/main/java/core/basesyntax/service/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriter {
```

Comment:
Make this interface more functional for future with changing names to more various, like Writer, write